src/pages/gateway/cicd.md

```diff
@@ -0,0 +1,242 @@
+
+After [creating a local environment](./developer-tools.md#create-a-local-environment) by running the `aio api-mesh:init` command, upload your local files to [GitHub](https://docs.github.com/en/repositories/creating-and-managing-repositories/quickstart-for-repositories).
+
+<InlineAlert variant="info" slots="text"/>
+
+When running the `aio api-mesh:init` command, ensure that you enable `git` by following the command prompts.
+
+Once your repository is accessible, clone your repo locally using the following command:
+
+```bash
+git clone https://github.com/<org>/<project_name>.git
+```
+
+You will continue to use this repo to make changes to your mesh.
+
+## 3. Configure your secrets and variables
+
+When developing locally, you can store your variables and secrets in your [environment variables file](./developer-tools.md#environment-variables). When using GitHub, you will need to use [secrets](https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions) and [variables](https://docs.github.com/en/actions/learn-github-actions/variables).
+
+- Variables are for information that is not sensitive, for example, a variable could determine if you are targeting `stage` or `prod`. Additionally, you can configure variables to carry different values based on the environment you are using.
+
+- Secrets are for sensitive information, such as access tokens. These are never exposed in the Github code repo.
+
+### Add a secret or variable
+
+<InlineAlert variant="info" slots="text"/>
+
+You need administrative permissions to the target GitHub repository to add secrets or variables.
+
+1. Navigate to `https://github.com/<org>/<project_name>/settings/secrets/actions`.
+
+1. On the Secrets or Variables tab, click either **New repository secret** or **New repository variable**.
+
+1. Enter a name for the secret or variable, such as `CLIENTID_PROD`.
+
+1. Enter the corresponding value for the secret or variable, for example, `123456123456`.
+
+The previously described `yml` file in [Create an action](#3-create-a-github-action) requires the following secrets/variables.
+
+```yml
+CLIENTID: ${{ secrets.CLIENTID_PROD }}
+CLIENTSECRET: ${{ secrets.CLIENTSECRET_PROD }}
+TECHNICALACCOUNTID: ${{ secrets.TECHNICALACCID_PROD }}
+TECHNICALACCOUNTEMAIL: ${{ secrets.TECHNICALACCEMAIL_PROD }}
+IMSORGID: ${{ secrets.IMSORGID_PROD }}
+ORGID: ${{ secrets.ORGID_PROD }}
+PROJECTID: ${{ secrets.PROJECTID_PROD }}
+WORKSPACEID: ${{ secrets.WORKSPACEID_PROD }}
+```
+
+For example, if you named a secret `CLIENTID_PROD`, you can call that secret using the `${{ secrets.CLIENTID_PROD }}` format.
+
+The specific secrets required for this workflow are available in the [Developer Console](https://console.adobe.io/) by navigating to the desired workspace within a project and clicking the **Download all** button in the top-right of the screen. This downloads a `JSON` file.
```

Comment:
Is `https://console.adobe.io/` the preferred URL? I thought that domain was replaced with `developer.adobe.com`.